src/app/lib/config/services/arabic.js

```diff
@@ -61,8 +61,9 @@ const service = {
             'أن تنقر زر التجديد (refresh) في برنامج التصفح الذي تستخدمه',
```

Comment:
Is there a reason why `refresh` isn't translated?

Reply:
Not 100% certain, but it seems that `refresh` is one of those words (like e.g. `cookies`) that are deliberately not translated for services. This is the case not only for Arabic, but for some other services too (e.g. Persian).

Reply:
Okay cool